tests/subsys/settings/fcb_init/prj.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_ZTEST=y
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_FLASH=y
+CONFIG_SOC_FLASH_NRF=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_ARM_CORE_MPU=n
+CONFIG_ARM_MPU=n
+CONFIG_ARM_MPU_NRF52X=n
+CONFIG_FCB=y
+
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_FCB=y
+CONFIG_SETTINGS_FCB_FLASH_AREA=3
```

Comment:
Since we now have the storage_partition that gets automatically enabled when CONFIG_FCB=y, I think you could use 4 here, or just leave this line out since 4 is the default value.